locales/hu_HU/LC_MESSAGES/duckduckgo.po

```diff
@@ -4579,14 +4567,16 @@ msgstr ""
 msgctxt "feedback form"
 msgid "We use feedback like this to improve DuckDuckGo. It really helps."
 msgstr ""
+"Ilyen visszajelzéseket használunk a DuckDuckGo fejlesztéséhez. Nagyon sokat "
+"segít."
 
 msgid ""
 "We'll block hidden trackers and encrypt connections to keep you more private."
 msgstr ""
 
 msgctxt "new user poll"
 msgid "We're honored to have you on the Duck Side"
-msgstr "Becsüljük, hogy a Kacsa Oldalon állsz"
+msgstr "Becsüljük, hogy a Kacsa oldalon állsz"
```

Comment:
We would prefer to leave `Duck` as is, because it's part of our brand. Does it still sound ok if you used `Duck Oldalon`?